share/spack/gitlab/cloud_pipelines/stacks/e4s-cray-rhel/spack.yaml

```
@@ -32,6 +32,8 @@ spack:
     elfutils:
       variants: +bzip2 ~nls +xz
       require: "%gcc"
+    gcc-runtime:
+      require: "%gcc"
```

why's this required if the package already requires `%gcc`?

---

It conflicts with the requirement on the compiler in `all`

---

Can you add a note in the YAML for this?

```suggestion
    # The gcc-runtime package requires %gcc.
    # This avoids the conflict between gcc-runtime and all:require:%cce
    gcc-runtime:
      require: "%gcc"
```

Also now wondering if we need `require_where_unspecified:` or something for this, as I feel like it'll be a fairly common little exception people have to deal with.

---

notice there are already multiple of these exceptions in the yaml file without comments

---

See https://github.com/spack/spack/pull/41806

---

If we have an issue that seems sufficient